build_rules.gradle

```diff
@@ -593,6 +593,118 @@ ext.applyJavaNature = {
   }
 }
 
+// Reads and contains all necessary performance test parameters
+class JavaPerformanceTestConfiguration {
+  String pkbLocation = System.getProperty('pkbLocation')
+
+  String logLevel = System.getProperty('logLevel', 'INFO')
+  String gradleBinary = System.getProperty('gradleBinary', './gradlew')
+  String isOfficial = System.getProperty('official', 'true')
```

Comment:
Do we want to default to `true` here?

Reply:
No we shouldn't - this would mark benchmark results being marked as official which even in Perfkit is defaulted to false. The task is for running them from local machines so those are definitely not official test results.  It should be false then - my bad.

I added a comment describing this in more detail in next commit